src/peft/peft_model.py

```diff
@@ -749,22 +749,30 @@ def get_prompt(
                 post_process_fn = TRANSFORMERS_MODELS_TO_PREFIX_TUNING_POSTPROCESS_MAPPING[self.config.model_type]
                 past_key_values = post_process_fn(past_key_values)
             elif ("gemma2" in model_type) or ("gemma3_text" in model_type):
+                # TODO: remove this logic once transformers < 4.56 is dropped
+                transformers_le_4_55 = packaging.version.parse(transformers.__version__) <= packaging.version.parse(
+                    "4.55.2"
+                )
```

Comment:
I agree it would make more sense to make it `lt_4_60` except I'm overlooking something.